app/controllers/api/v1/publications_controller.rb

```diff
@@ -226,6 +238,72 @@ def grants
           key :api_key, []
         end
       end
+
+      operation :patch do
+        key :summary, 'Update Publications'
+        key :description, 'Update publication\'s ScholarSphere Open Access Link'
+        key :operationId, 'updatePublications'
+        key :produces, [
+          'application/json',
+          'text/html'
+        ]
+        key :tags, [
+          'publication'
+        ]
+        parameter do
+          key :name, :Publications
+          key :in, :body
+          key :description, 'Publications to update with the provided ScholarSphere Open Access Link'
+          key :required, true
+          schema do
+            key :'$ref', :PublicationCollectionInput
+          end
+        end
+        response 200 do
+          key :description, 'publications update success response'
+          schema do
+            key :'$ref', :PublicationsPatchResult
+          end
+        end
+        response 404 do
+          key :description, 'unauthorized'
```

Comment:
I think this should be 'not_found' or something along those lines.

Reply:
yes that is right, nice catch! I will fix that.

Reply:
fixed with the last commit